### DISCUSSION TITLE
Run as a replica set?

### DISCUSSION THREAD

**Discussion by kakarot-dev** [prisma/prisma#12649] [Q&A] [answered]
What is the meaning of this error?
![image](https://user-images.githubusercontent.com/76399483/161421087-9d5019ed-3e22-4608-b9dc-03f8fe595f16.png)
My format is 
```
datasource db {
  provider = "mongodb"
  url      = "mongodb://root:Password123@ip:port/project?authSource=admin&readPreference=primary&appname=MongoDB%20Compass&ssl=false"
}
```

**Comment by austincrim** [comment, marked as answer]
Hey @kakarot-dev 👋 ,

Prisma uses MongoDB transactions in its implementation which requires a [MongoDB replica set](https://www.mongodb.com/docs/manual/replication/). [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) is the easiest way to get a replica set running, but it's possible to [set one up locally](https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set) as well.

We talk about this a little [in our docs](https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set).

Let me know if I can clarify anything!

**Reply by kakarot-dev** [reply]
Ah, thx for the links, helped a lot!!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
